app/Http/Controllers/CategoriaController.php

```diff
@@ -12,8 +12,9 @@ class CategoriaController extends Controller
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(Request $request)
     {
+        if (!$request->ajax()) return redirect('/');
         $categorias = Categoria::all();
         return $categorias;
```

Comment:
Make this in only one line 
```suggestion
        return Categoria::all()
```